File: Android/Pruebas/app/src/main/java/com/example/prueba/MainActivity.java
package com.example.prueba;

import android.graphics.drawable.Drawable;
import android.media.MediaPlayer;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.Button;
import android.widget.CheckBox;
import android.widget.CompoundButton;
import android.widget.MediaController;
import android.widget.RadioButton;
import android.widget.RadioGroup;
import android.widget.Spinner;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;
import android.view.View.OnClickListener; //Importante agregarlo
import android.app.Activity;
import android.widget.ToggleButton;
import android.widget.VideoView;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState); setContentView(R.layout.activity_main);
        WebView wv = (WebView)findViewById(R.id.webViewUser); wv.loadUrl("https://www.google.com.mx/?hl=es");
    } }

File: Android/pruebasKotlin/app/src/main/java/com/example/pruebakotlin/MainActivity.kt
package com.example.pruebakotlin

import android.content.Context
import android.content.Intent
import android.content.SharedPreferences
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import java.io.BufferedReader
import java.io.FileNotFoundException
import java.io.FileOutputStream
import java.io.IOException
import java.io.InputStreamReader

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val btnLoad: Button = findViewById(R.id.button2)
        val tv: TextView = findViewById(R.id.tvLoadData)

        btnLoad.setOnClickListener {
            val eol = System.getProperty("line.separator")

            try {
                val input = BufferedReader(InputStreamReader(openFileInput("test_file")))
                val buffer = StringBuffer()
                var line: String?

                while (input.readLine().also { line = it } != null) {
                    buffer.append(line).append(eol)
                }

                tv.text = buffer.toString()
            } catch (e: FileNotFoundException) {
                e.printStackTrace()
            } catch (e: IOException) {
                e.printStackTrace()
            }
        }
    }
}